docs/DeveloperGuide.md

```diff
@@ -8,23 +8,128 @@
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+## Implementation
+
```

Comment:
You could add the system architecture diagram in the next iteration of this DG to depict the high-level design of your software and make it more detailed.